src/libopensc/reader-pcsc.c

```diff
@@ -303,7 +304,8 @@ static int pcsc_internal_transmit(sc_reader_t *reader,
 
 static int pcsc_transmit(sc_reader_t *reader, sc_apdu_t *apdu)
 {
-	size_t ssize, rsize, rbuflen = 0;
+	size_t ssize, rbuflen = 0;
+	DWORD rsize;
```

Comment:
rsize is used as input to pcsc_internal_transmit. all other instances allow implicit casts. For pcsc_internal_transmit, however, size_t is required as type

Reply:
I probably do not follow. Can you explain more?
```
static int
pcsc_internal_transmit(sc_reader_t *reader,
		const u8 *sendbuf, DWORD sendsize,
		u8 *recvbuf, DWORD *recvsize,
		unsigned long control)
{
...
	DWORD dwSendLength, dwRecvLength;
...
	dwSendLength = sendsize;
	dwRecvLength = *recvsize;
...
		rv = priv->gpriv->SCardTransmit(card, &sSendPci, sendbuf, dwSendLength,
				   &sRecvPci, recvbuf, &dwRecvLength);
...
typedef LONG (PCSC_API *SCardTransmit_t)(SCARDHANDLE hCard, LPCSCARD_IO_REQUEST pioSendPci,
	LPCBYTE pbSendBuffer, DWORD cbSendLength, LPSCARD_IO_REQUEST pioRecvPci,
```

Reply:
sorry, I missed the change here which also modifies the signature of pcsc_internal_transmit to accept a DWORD.

However, I believe the implicit approach here historically was that pcsc_internal_transmit offers only types that are generic (i.e. not specific to winscard.dll). I think the better approach would be to convert those types within pcsc_internal_transmit instead of changing the function's signature.